python/servo/platform/__init__.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2023 The Servo Project Developers. See the COPYRIGHT
+# file at the top-level directory of this distribution.
+#
+# Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+# http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+# <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+# option. This file may not be copied, modified, or distributed
+# except according to those terms.
+
+import platform
+
+from .linux import Linux
```

Comment:
The builds for mac and windows are failing because importing linux will cause 'distro' to be imported, which I believe is not installed on those platforms.

I guess we can move these imports into their respective if clauses in the `get()` function.

Reply:
Yep, that's definitely the issue. I've moved to making the imports conditional in `__init__.py` instead as I think that might make the platform-specific code a little less brittle.